suite-native/transactions/src/components/TransactionDetail/TransactionDetailAddressesSection.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { TouchableOpacity } from 'react-native-gesture-handler';
+
+import { Box, Text } from '@suite-native/atoms';
+import { AccountAddressFormatter } from '@suite-native/formatters';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { CryptoIcon, CryptoIconName } from '@trezor/icons';
+
+import { SummaryRow } from './TransactionDetailStatusSection';
+
+type TransactionDetailAddressesSectionProps = {
+    addresses: string[];
+    addressesType: 'inputs' | 'outputs';
+    onShowMore: () => void;
+    cryptoIcon?: CryptoIconName;
+};
+
+const showMoreButtonStyle = prepareNativeStyle(_ => ({ flexDirection: 'row' }));
+
+const hiddenTransactionsCountStyle = prepareNativeStyle(utils => ({
+    justifyContent: 'center',
+    marginLeft: utils.spacings.small,
+    backgroundColor: utils.colors.backgroundNeutralSubtleOnElevation1,
+    borderRadius: utils.borders.radii.round,
+    paddingHorizontal: utils.spacings.small,
+    paddingVertical: utils.spacings.small / 4,
+}));
+
+const addressTextStyle = prepareNativeStyle(() => ({
```

Comment:
```suggestion
const addressTextStyle = prepareNativeStyle(_ => ({
```

Does typescript not scream about it?

Reply:
It is valid not to use any parameters at all. The repo is full of this. I've changed it anyway in 8e7e8f721 to make it the same as the other style objects of this file.